setup.py

```diff
@@ -25,7 +25,8 @@
         'dev': [
             'pytest>=4.6.9',
             'coverage~=5.0.4',
-            'sqlalchemy-utils~=0.36.3',
+            # for sqlalchemy1.4 >= 0.37 is required
+            'sqlalchemy-utils>=0.36.3',
```

Comment:
`>= 3.7` != `>-0.36.3`

Reply:
That was intended. py27 wasn't compatible with `0.36.3`, so that customisation was done in `tox.ini`. Anyway, I've removed py27 support now.